_install-and-configure/install-opensearch/docker.md

```diff
@@ -31,6 +31,7 @@ If you need to install Docker Compose manually and your host supports Python, yo
 
 ## Important host settings
 
```

Comment:
This need an introductory sentence to fix the Vale error. Could be something like...

```suggestion
Based on your host system, certain settings must be configured in order to install OpenSearch using Docker.
```